### PR TITLE
update WIS guide Part 5

### PR DESCRIPTION
**Pull request by tomkralidis**
As per #92

### CODE REVIEW

File: docs/guide/000-introduction.adoc [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-- Additional examples of WCMP metadata (for particular product types): https://wis.wmo.int/MD-Examples.
-
+- WCMP 1.3 Template XML records: Template records containing placeholders (to be replaced with information related to the specific product described by the WIS discovery metadata record) are available from https://old.wmo.int/wiswiki/tiki-index.php%3Fpage=MDG_Guide.html. A valid example of an XML record, with field content (to be replaced) is also available there;
+- WIS Wiki Page on WIS discovery metadata: https://community.wmo.int/activity-areas/wis/wcmp;
+- WCMP documentation: 
+* Part 1 - Conformance Requirements provides the basic definition of the Core Profile : http://wis.wmo.int/WCMPpart1; 
+* Part 2 - Abstract Test Suite, Data Dictionary and Code Lists describes the constraints on how metadata may be written and defines the definitions of terms used within the metadata : http://wis.wmo.int/WCMPPart2;
+- Additional guidance WCMP documentation: https://old.wmo.int/wiswiki/tiki-index.php%3Fpage=IndexForMetadataGuidance.html ;
+- Additional examples of WCMP metadata (for particular product types): https://old.wmo.int/wiswiki/tiki-index.php%3Fpage=MDG_EG.html
```
Comment (jsieland) [inline docs/guide/000-introduction.adoc:15]: @amilan17 please check links - thx!

File: docs/guide/003-compliance-with-additional-metadata-standards.adoc [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 
-Each ISO 19115 profile defines specific rules that should be met. For example, to comply with the INSPIRE metadata profile, the additional requirements to be met include the provision of one keyword from the general environmental multilingual thesaurus (GEMET), a lineage statement and a statement of conformance with European Commission Regulation (EC) No. 1205/2008.
+Examples of type (a) ISO 19115 profiles, in addition to the WCMP, include the _Infrastructure for Spatial Information in the European Community (INSPIRE)_ Metadata Profile, the North American Profile, the _Australian and New Zealand Information Council (ANZLIC)_ Metadata Profile and UK GEMINI. An example of (b) is the Marine Community Profile. For more information see also http://www.dcc.ac.uk/resources/metadata-standards/iso-19115.
```
Comment (jsieland) [inline docs/guide/003-compliance-with-additional-metadata-standards.adoc:8]: @amilan17 please check link - thx!

File: docs/guide/003-compliance-with-additional-metadata-standards.adoc [hunk 3]
```diff
@@ -1,12 +1,13 @@
-== Compliance with Additional Metadata Standards
+== Compliance with Metadata Standards
 
 This part of the Guide provides information to help create metadata records that comply with WCMP 1.3. This profile is based on ISO 19115 which provides two profiling mechanisms:
 [loweralpha]
 . A more constrained use of ISO 19115 (either by recommending use of fewer fields, making an optional element mandatory, or constraining the expected content of a field) to suit the needs of a particular community;
 . In addition to (a), the possibility of defining additional non-ISO 19115 fields (and field content) to be added to any record.
-Examples of type (a) ISO 19115 profiles, in addition to the WCMP, include the Infrastructure for Spatial Information in the European Community (INSPIRE) Metadata Profile, the North American Profile, the Australian and New Zealand Information Council (ANZLIC) Metadata Profile and UK GEMINI. An example of (b) is the Marine Community Profile. For more information see also http://www.dcc.ac.uk/resources/metadata-standards/iso-19115.
```
Comment (jsieland) [inline docs/guide/003-compliance-with-additional-metadata-standards.adoc:7]: @amilan17 please check link - thx!

File: docs/guide/005-principles-of-metadata-management-in-the-wis.adoc [hunk 4]
```diff
@@ -17,5 +17,5 @@ How to publish metadata
 
 Note: For a limited number of records, it is also possible to use the online editing services of a GISC.
 
-For more comprehensive information regarding the WIS and publishing metadata on the WIS, please consult the Manual on WIS (https://wis.wmo.int/WIS-Manual).
+For more comprehensive information regarding the WIS and publishing metadata on the WIS, please consult the Manual on WIS (https://community.wmo.int/activity-areas/wis/wis-guides).
```
Comment (jsieland) [inline docs/guide/005-principles-of-metadata-management-in-the-wis.adoc:20]: @amilan17 please check link - thx!

File: docs/guide/007-necessary-information-to-create-wcmp.adoc [hunk 5]
```diff
@@ -119,7 +124,7 @@ Below are typical abstracts and titles for:
...
 
-More examples of metadata titles and abstracts can be found in the WIS Wiki at http://wis.wmo.int/MD-Examples.
+More examples of metadata titles and abstracts can be found in the WIS Wiki at https://old.wmo.int/wiswiki/tiki-index.php%3Fpage=MDG_EG.html.
```
Comment (jsieland) [inline docs/guide/007-necessary-information-to-create-wcmp.adoc:127]: @amilan17 please check link - thx!

File: docs/guide/007-necessary-information-to-create-wcmp.adoc [hunk 6]
```diff
@@ -604,13 +614,14 @@ a|
 
 |===
 
-Any WCMP metadata record shall have at least one WMO_CategoryCode keyword, and the related KeywordTypeCode will be “theme”.
+Any WCMP metadata record shall have at least one `WMO_CategoryCode` keyword, and the related `KeywordTypeCode` will be `"theme"`.
 
-The WMO_CategoryCode list of terms is occasionally revised. For the latest list of terms, see: http://wis.wmo.int/2012/codelists/WMOCodeLists.xml#WMO_CategoryCode.
+The `WMO_CategoryCode` list of terms is occasionally revised. For the latest list of terms, see: http://wis.wmo.int/2012/codelists/WMOCodeLists.xml#WMO_CategoryCode.
```
Comment (jsieland) [inline docs/guide/007-necessary-information-to-create-wcmp.adoc:619]: @amilan17 please check link - thx!

File: docs/guide/008-technical-document.adoc [hunk 7]
```diff
@@ -1,4 +1,4 @@
 == Technical Document
 
-More details on the WCMP metadata can be found at http://wis.wmo.int/MD_Index.
+More details on the WCMP metadata can be found at https://community.wmo.int/activity-areas/wis/wcmp.
```
Comment (jsieland) [inline docs/guide/008-technical-document.adoc:3]: @amilan17 please check link - thx!

File: docs/guide/003-compliance-with-additional-metadata-standards.adoc (hunk 2 above)
Comment (amilan17) [reply docs/guide/003-compliance-with-additional-metadata-standards.adoc:8]: This URL is ok.

File: docs/guide/000-introduction.adoc (hunk 1 above)
Comment (amilan17) [reply docs/guide/000-introduction.adoc:15]: I think we should remove all of these URLs, except for the link to the WCMP homepage on the Community site.

Comment (amilan17) [reply docs/guide/000-introduction.adoc:15]: I will work on the draft for this section.

File: docs/guide/004-wcmp-validation-tools.adoc [hunk 8]
```diff
@@ -6,17 +6,17 @@ A set of ISO and WCMP validation tools can be used to ensure that a created WCMP
...
 
-Below is a list of web services and tools used to validate WCMP 1.3 and ISO 19115/19139 metadata records.
+Below is a list of web services and tools used to validate WCMP 1.3 and ISO 19115/19139 metadata records. As per section 5.4.3 other validation tools may be required for validating additional standards. 
 
 === WCMP 1.3 validation services and tools:
 
 . NOAA's WMO validation service: https://www.ngdc.noaa.gov/docucomp/validationServicesWmo;
 GeoNetwork-ANZMEST, with WCMP validation tool: https://sourceforge.net/projects/anzmest/files/bom-releases/.  This directory contains the Australian Bureau of Meteorology releases of ANZMEST 2.10.x (based on GeoNetwork), which include the WCMP 1.3 editing and validation tool.  For instructions on running the software and validation tool, see the WIS Wiki page on validation tools below;
-. WIS Wiki page on validation tools: http://wis.wmo.int/MD-Validate.
+. WIS Wiki page on validation tools: https://old.wmo.int/wiswiki/tiki-index.php%3Fpage=MDG_Test.html
```
[the remark this answers is not in the document]
Comment (amilan17) [reply docs/guide/004-wcmp-validation-tools.adoc:19]: The NOAA link is broken. I think we should move the links to working validation tools to the Community site.

File: docs/guide/005-principles-of-metadata-management-in-the-wis.adoc (hunk 4 above)
Comment (amilan17) [reply docs/guide/005-principles-of-metadata-management-in-the-wis.adoc:20]: this is the official link to the Manual on WIS: https://library.wmo.int/index.php?lvl=notice_display&id=9254

File: docs/guide/007-necessary-information-to-create-wcmp.adoc (hunk 5 above)
Comment (amilan17) [reply docs/guide/007-necessary-information-to-create-wcmp.adoc:127]: I think we should remove this sentence.

File: docs/guide/007-necessary-information-to-create-wcmp.adoc (hunk 6 above)
Comment (amilan17) [reply docs/guide/007-necessary-information-to-create-wcmp.adoc:619]: this URL is good.

File: docs/guide/007-necessary-information-to-create-wcmp.adoc [hunk 9]
```diff
@@ -1119,25 +1135,25 @@ Ordnance Survey Open Data License [https://www.ordnancesurvey.co.uk/docs/licence
...
-The scope of distribution should, ideally, be added as a keyword using the term “OriginatingCentre”.
+The scope of distribution should, ideally, be added as a keyword using the term `"OriginatingCentre"`.
 
 Please refer to the encoding of scope of distribution, provided under Example 1 above or in section 5.8.1.8.2.
 
-Example 3: GTS data intended for global exchange
+===== Example 3: GTS data intended for global exchange
 
-This example describes data distributed via the GTS and available from the cache at a GISC. For data delivered via the GTS, the data policy term to be added to the “otherConstraints” field can only be “WMOAdditional” or “WMOEssential” – both of these terms are defined at http://wis.wmo.int/2012/codelists/WMOCodeLists.xml#WMO_DataLicenseCode.
+This example describes data distributed via the GTS and available from the cache at a GISC. For data delivered via the GTS, the data policy term to be added to the `otherConstraints` field can only be `"WMOAdditional"` or `"WMOEssential"` – both of these terms are defined at http://wis.wmo.int/2012/codelists/WMOCodeLists.xml#WMO_DataLicenseCode.
```
[the remark this answers is not in the document]
Comment (amilan17) [reply docs/guide/007-necessary-information-to-create-wcmp.adoc:1145]: this link is good

File: docs/guide/008-technical-document.adoc (hunk 7 above)
Comment (amilan17) [reply docs/guide/008-technical-document.adoc:3]: I think we should delete this section. We already have a link at the beginning to this URL.

**Comment by amilan17** [comment]
URLs in XML examples also need to be updated. 

old: [http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/resources/Codelist/gmxCodelists.xml](http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/resources/Codelist/gmxCodelists.xml) 

new: [https://standards.iso.org/iso/19139/resources/gmxCodelists.xml](https://standards.iso.org/iso/19139/resources/gmxCodelists.xml)

File: docs/guide/008-technical-document.adoc (hunk 7 above)
Comment (josusky) [reply docs/guide/008-technical-document.adoc:3]: Well, this is the only sentence in the "008-technical-document.adoc". We can remove the whole document (chapter then).

**Review by josusky — APPROVED** [review]

File: docs/guide/004-wcmp-validation-tools.adoc (hunk 8 above)
[the remark this answers is not in the document]
Comment (amilan17) [reply docs/guide/004-wcmp-validation-tools.adoc]: I removed this section

File: docs/guide/007-necessary-information-to-create-wcmp.adoc (hunk 5 above)
Comment (amilan17) [reply docs/guide/007-necessary-information-to-create-wcmp.adoc]: removed

File: docs/guide/008-technical-document.adoc (hunk 7 above)
Comment (amilan17) [reply docs/guide/008-technical-document.adoc]: removed chapter

**Comment by tomkralidis** [comment]
TT-WISMD 2021-07-02: agree to merge and review corresponding Word Document.  Great work all!

**Pull request merged**